udemy_exercises/songs/Song.java

```diff
@@ -1,5 +1,19 @@
-package udemy_exercises.songs;
 
-class Song {
+public class Song {
+    private String title;
+    private double duration;
 
+    public Song(String title, double duration) {
+        this.title = title;
+        this.duration = duration;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Excellent implementation!!  You have leveraged core Java functionality to provide a clear object description with minimal (if any) code on the part of the client.  Big Bonus points here!!